examples/file/README.md

```diff
@@ -0,0 +1,31 @@
+SystemLink File Service API Examples
+====================================
+
+This directory holds self-contained example projects for the SystemLink File service API. To make finding relevant examples easier, a summary of each one is listed below.
+
+Examples for other SystemLink Client APIs are available in the
+[root examples directory](..).
+
+Running an Example
+------------------
+
+Unless otherwise specified by the example's README, each example is run in the
+same way:
```

Comment:
There was a note I had in one of these reviews about making this boilerplate a bit clearer -- something like "Complete the following steps unless the readme specifies otherwise" -- but I can't find it for the life of me and it's not high impact. But if that rings any bells, the same note applies here.